test/e2e/network/dns.go

```diff
@@ -128,19 +161,19 @@ func createProbeCommand(namesToResolve []string, hostEntries []string, ptrLookup
 	for _, name := range hostEntries {
 		fileName := fmt.Sprintf("%s_hosts@%s", fileNamePrefix, name)
 		fileNames = append(fileNames, fileName)
-		probeCmd += fmt.Sprintf(`test -n "$$(getent hosts %s)" && echo OK > /results/%s;`, name, fileName)
+		probeCmd += fmt.Sprintf(`test -n "$$(getent %s %s)" && echo OK > /results/%s;`, getentdb, name, fileName)
 	}
 
-	podARecByUDPFileName := fmt.Sprintf("%s_udp@PodARecord", fileNamePrefix)
-	podARecByTCPFileName := fmt.Sprintf("%s_tcp@PodARecord", fileNamePrefix)
-	probeCmd += fmt.Sprintf(`podARec=$$(hostname -i| awk -F. '{print $$1"-"$$2"-"$$3"-"$$4".%s.pod.cluster.local"}');`, namespace)
-	probeCmd += fmt.Sprintf(`test -n "$$(dig +notcp +noall +answer +search $${podARec} A)" && echo OK > /results/%s;`, podARecByUDPFileName)
-	probeCmd += fmt.Sprintf(`test -n "$$(dig +tcp +noall +answer +search $${podARec} A)" && echo OK > /results/%s;`, podARecByTCPFileName)
+	podARecByUDPFileName := fmt.Sprintf("%s_udp@Pod%sRecord", fileNamePrefix, typeA)
+	podARecByTCPFileName := fmt.Sprintf("%s_tcp@Pod%sRecord", fileNamePrefix, typeA)
+	probeCmd += fmt.Sprintf(`podARec="$$(hostname -i| awk -F. '{print $$1"-"$$2"-"$$3"-"$$4".%s.pod.cluster.local"}')";`, namespace)
```

Comment:
There is an error here.
I did not take the right file. should call the ReverseAddr function